File: TabBarController_no_stroyboard_objective-c/TabBarController_no_storyboard_obj-c/MyViewController.m
//
//  ViewController.m
//  TabBarController_no_storyboard_obj-c
//
//  Created by SeungYeon Yoo on 2022/04/24.
//

#import "MyViewController.h"

@interface MyViewController ()

@end

@implementation MyViewController

- (id) initWithParams:(NSString *)title bgColor:(UIColor *)bgColor {
    self.title = title;
    self.view.backgroundColor = bgColor;
    return self;
}

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}


@end

File: TabBarController_withoutStoryboard-/TabBarController_No_storyboard_swift/MyViewController.swift
//
//  ViewController.swift
//  TabBarController_No_storyboard_swift
//
//  Created by SeungYeon Yoo on 2022/04/24.
//

import UIKit

class MyViewController: UIViewController {
    
    convenience init(title: String, bgColor: UIColor) {
        self.init()
        self.title = title
        self.view.backgroundColor = bgColor
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }


}

